core/controllers/base.py

```diff
@@ -195,6 +195,8 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         self.is_super_admin = (
             current_user_services.is_current_user_super_admin())
 
+        self.values['additional_angular_modules'] = []
+        self.values['iframed'] = None
```

Comment:
This is usually True or False, probably start with False and let subhandlers override?

Reply:
Yeah! I was too thinking this, thanks for adding.

Reply:
Done!